src/components/PublicityHeader/index.tsx

```diff
@@ -1,16 +1,36 @@
 import { useState } from 'react';
 import Slider from 'components/Slider';
 import { ITEMS } from 'constants/constants';
+import { BANNER, SERVICES } from './constants';
 
 import styles from './styles.module.scss';
 
 function PublicityHeader() {
   const [auto, setAuto] = useState(false);
   return (
-    <div>
-      <Slider hide auto={auto} setAuto={setAuto} slides={ITEMS} className={styles.image} />
+    <>
+      <div>
+        <Slider hide auto={auto} setAuto={setAuto} slides={ITEMS} className={styles.image} />
+      </div>
+      <div className={styles.services}>
+        {SERVICES.map((service) => (
+          <div className={styles.iconsSection}>
```

Comment:
Move this block of code to a new component

Reply:
done